filebeat/filebeat.full.yml

```diff
@@ -510,6 +510,14 @@ output.elasticsearch:
   # Path to the Elasticsearch 2.x version of the template file.
   #template.versions.2x.path: "${path.config}/filebeat.template-es2x.json"
 
+  # If set to true, filebeat checks the Elasticsearch version at connect time, and if it
+  # is 5.x, it loads the file specified by the template.versions.5x.path setting. The
+  # default is true.
+  #template.versions.5x.enabled: true
+
+  # Path to the Elasticsearch 5.x version of the template file.
+  #template.versions.5x.path: "${path.config}/filebeat.template-es5x.json"
```

Comment:
See my comment below about the naming. For the next months until 6 is GA, loading the 5.x is going to be the expected default. I'm thinking if we should have path config now for all versions? Problem is that we add even more config options (which will be remove afterwards) :-( WDYT?